### DISCUSSION TITLE
Add contributing guidelines

### DISCUSSION THREAD

**Discussion by GidonFrischkorn** [venpopov/bmm#192] [Ideas]
I saw that GitHub recommends to write contributing guidelines to lay out what is considered a sufficient contribution to be listed as a contributor. I think this would be a good idea, to give potential contributors an idea what would be expected from them to have their contributions visible for the package.

I will just start a list and maybe we can collect contributions that would pass this threshold.

Level of contributions for becoming a contributor:
- Add a new model to `bmm` including documentation (vignette, parameter recovery, testing)
- Contribute base functionality that eases the use of all models (e.g. plotting functions, data preparation, post-processing of results)
- Contribute functionality that integrates important functionality from related packages (e.g. `emmeans`, `tidybayes`, `bayesplot`, etc.) to `bmm`

Levels of contributions mentioned in the `NEWS`
- bug fixes to models or core functionality in `bmm`
- contributions to improve stability or speed of estimating `bmmodels`

This is just a start, and please do post suggestions to clarify these contributions.

**Comment by venpopov** [comment]
Looks good! Great idea to have this for transparency

**Comment by GidonFrischkorn** [comment]
Perfect, then I can take care of this.
